ui/command_buffer_model.cpp

```diff
@@ -419,7 +420,7 @@ bool CommandBufferModel::CreateNodeToParentMap(uint64_t parent_row,
         // one of the children. Will no longer be part of current event/pass in the future If
         // is_selected==false, and return value is true, then that means event/pass started in one
         // of the children. Will continue be part of the current event/pass going forward
-        is_selected = CreateNodeToParentMap(child, child_node_index, is_selected);
+        is_selected = CreateNodeToParentMap(num_children + child, child_node_index, is_selected);
```

Comment:
Does it mean that we will have more nodes now? :p

Reply:
This is a bug fix that took me forever to track down.
Basically the order of entries in each level of the tree is the normal children followed by the shared children. 

For example:
![image](https://github.com/user-attachments/assets/d8201f07-9ce4-4275-b4cb-683cd9afe961)

Here we see ADDR_LO, ADDR_HI, and SIZE as field nodes, before a whole bunch of shared nodes appear. So the first shared node, CP_SET_DRAW_STATE, is actually in the 4th row. Without the "num_children+" part, CP_SET_DRAW_STATE is set to the in the 0th row, which causes all sorts of seemingly random issues with QT.

Reply:
good to know, thanks!